packages/patternfly-4/react-core/src/components/Dropdown/DropdownItem.tsx

```diff
@@ -23,6 +23,8 @@ export interface DropdownItemProps extends InternalDropdownItemProps {
   tooltip?: React.ReactNode;
   /** Additional tooltip props forwarded to the Tooltip component */
   tooltipProps?: any;
+  /** Additional node to include alongside item within the <li> */
+  additionalChildren?: React.ReactNode;
```

Comment:
Do we really need this prop?  I am not understanding why AplicationLauncher can't create a single child for the dropdownItem.  Is it because it would be 2 fragments and not a single node?  If so, can we wrap it in `<React.Fragment>`?

Reply:
This is needed to match the html structure in core, and prevents nesting buttons inside anchors/buttons (if it was a child of the dropdown).